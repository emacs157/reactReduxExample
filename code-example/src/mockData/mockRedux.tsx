import { ReduxState } from '../assets/interface';

export const mockRedux: ReduxState = {
    Prefectures: {
        status: 'success',
        results: [
            {
                prefCode: 1,
                prefName: '北海道',
            },
            {
                prefCode: 2,
                prefName: '青森県',
            },
            {
                prefCode: 3,
                prefName: '岩手県',
            },
            {
                prefCode: 4,
                prefName: '宮城県',
            },
            {
                prefCode: 5,
                prefName: '秋田県',
            },
            {
                prefCode: 6,
                prefName: '山形県',
            },
            {
                prefCode: 7,
                prefName: '福島県',
            },
            {
                prefCode: 8,
                prefName: '茨城県',
            },
            {
                prefCode: 9,
                prefName: '栃木県',
            },
            {
                prefCode: 10,
                prefName: '群馬県',
            },
            {
                prefCode: 11,
                prefName: '埼玉県',
            },
            {
                prefCode: 12,
                prefName: '千葉県',
            },
            {
                prefCode: 13,
                prefName: '東京都',
            },
            {
                prefCode: 14,
                prefName: '神奈川県',
            },
            {
                prefCode: 15,
                prefName: '新潟県',
            },
            {
                prefCode: 16,
                prefName: '富山県',
            },
            {
                prefCode: 17,
                prefName: '石川県',
            },
            {
                prefCode: 18,
                prefName: '福井県',
            },
            {
                prefCode: 19,
                prefName: '山梨県',
            },
            {
                prefCode: 20,
                prefName: '長野県',
            },
            {
                prefCode: 21,
                prefName: '岐阜県',
            },
            {
                prefCode: 22,
                prefName: '静岡県',
            },
            {
                prefCode: 23,
                prefName: '愛知県',
            },
            {
                prefCode: 24,
                prefName: '三重県',
            },
            {
                prefCode: 25,
                prefName: '滋賀県',
            },
            {
                prefCode: 26,
                prefName: '京都府',
            },
            {
                prefCode: 27,
                prefName: '大阪府',
            },
            {
                prefCode: 28,
                prefName: '兵庫県',
            },
            {
                prefCode: 29,
                prefName: '奈良県',
            },
            {
                prefCode: 30,
                prefName: '和歌山県',
            },
            {
                prefCode: 31,
                prefName: '鳥取県',
            },
            {
                prefCode: 32,
                prefName: '島根県',
            },
            {
                prefCode: 33,
                prefName: '岡山県',
            },
            {
                prefCode: 34,
                prefName: '広島県',
            },
            {
                prefCode: 35,
                prefName: '山口県',
            },
            {
                prefCode: 36,
                prefName: '徳島県',
            },
            {
                prefCode: 37,
                prefName: '香川県',
            },
            {
                prefCode: 38,
                prefName: '愛媛県',
            },
            {
                prefCode: 39,
                prefName: '高知県',
            },
            {
                prefCode: 40,
                prefName: '福岡県',
            },
            {
                prefCode: 41,
                prefName: '佐賀県',
            },
            {
                prefCode: 42,
                prefName: '長崎県',
            },
            {
                prefCode: 43,
                prefName: '熊本県',
            },
            {
                prefCode: 44,
                prefName: '大分県',
            },
            {
                prefCode: 45,
                prefName: '宮崎県',
            },
            {
                prefCode: 46,
                prefName: '鹿児島県',
            },
            {
                prefCode: 47,
                prefName: '沖縄県',
            },
        ],
    },
    Population: {
        status: 'success',
        requestParams: {
            prefName: '宮崎県',
            prefCode: 45,
            cityCode: '-',
        },
        results: [
            {
                name: '栃木県',
                year: [
                    1960, 1965, 1970, 1975, 1980, 1985, 1990, 1995, 2000, 2005,
                    2010, 2015, 2020, 2025, 2030, 2035, 2040, 2045,
                ],
                data: [
                    1513624, 1521656, 1580021, 1698003, 1792201, 1866066,
                    1935168, 1984390, 2004817, 2016631, 2007683, 1974255,
                    1930235, 1872842, 1805949, 1730414, 1647288, 1560619,
                ],
                young: [
                    510097, 429447, 389016, 414123, 433232, 425807, 380087,
                    339253, 306905, 285245, 269823, 252836, 234328, 215694,
                    200974, 186959, 175984, 164918,
                ],
                adult: [
                    909207, 986774, 1069584, 1143083, 1191109, 1243861, 1315228,
                    1350635, 1352311, 1336513, 1281274, 1203616, 1138540,
                    1084830, 1031754, 969256, 883575, 812961,
                ],
                elderly: [
                    94320, 105435, 121421, 140651, 167438, 196328, 238505,
                    292947, 344506, 390896, 438196, 508392, 557367, 572318,
                    573221, 574199, 587729, 582740,
                ],
            },
            {
                name: '高知県',
                year: [
                    1960, 1965, 1970, 1975, 1980, 1985, 1990, 1995, 2000, 2005,
                    2010, 2015, 2020, 2025, 2030, 2035, 2040, 2045,
                ],
                data: [
                    854595, 812714, 786882, 808397, 831275, 839784, 825034,
                    816704, 813949, 796292, 764456, 728276, 691090, 652757,
                    614449, 575728, 536443, 498460,
                ],
                young: [
                    253081, 200055, 171991, 173005, 173649, 168580, 144276,
                    125833, 111740, 102421, 92798, 83884, 76335, 68775, 62306,
                    56380, 52105, 48198,
                ],
                adult: [
                    528882, 530837, 524918, 536129, 547780, 548167, 535995,
                    522208, 509050, 487367, 447540, 400605, 370062, 343706,
                    319382, 296169, 263431, 237246,
                ],
                elderly: [
                    72632, 81822, 89973, 98752, 109116, 121759, 141508, 167967,
                    191729, 206375, 218148, 237012, 244693, 240276, 232761,
                    223179, 220907, 213016,
                ],
            },
            {
                name: '宮崎県',
                year: [
                    1960, 1965, 1970, 1975, 1980, 1985, 1990, 1995, 2000, 2005,
                    2010, 2015, 2020, 2025, 2030, 2035, 2040, 2045,
                ],
                data: [
                    1134590, 1080692, 1051105, 1085055, 1151587, 1175543,
                    1168907, 1175819, 1170007, 1153042, 1135233, 1104069,
                    1066719, 1023170, 976626, 928034, 876863, 824806,
                ],
                young: [
                    405950, 335056, 278242, 265987, 273766, 270084, 239738,
                    210806, 187431, 169075, 158588, 149608, 142582, 132392,
                    122197, 112605, 104925, 97973,
                ],
                adult: [
                    661023, 668656, 684091, 716077, 757074, 764617, 761368,
                    760751, 740401, 712527, 680854, 622544, 573889, 532963,
                    500317, 471452, 432511, 397299,
                ],
                elderly: [
                    67617, 76980, 88772, 102923, 120638, 140747, 166539, 204222,
                    241754, 270586, 291301, 322975, 350248, 357815, 354112,
                    343977, 339427, 329534,
                ],
            },
        ],
    },
    _persist: {
        version: 1,
        rehydrated: true,
    },
};
