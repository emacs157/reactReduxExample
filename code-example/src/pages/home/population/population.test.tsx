import configureStore from 'redux-mock-store';
import { mockRedux } from '../../../mockData/mockRedux';
import React from 'react';
import { Provider } from 'react-redux';
import PopulationComponent from './population';
import { render, screen } from '@testing-library/react';

const mockStore = configureStore();
const store = mockStore(mockRedux);

const thisComponent = (
	<Provider store={store}>
		<PopulationComponent population={mockRedux.Population} />
	</Provider>
);

it('matches snapshot', () => {
	const { asFragment } = render(thisComponent);
	expect(asFragment()).toMatchSnapshot();
});

it('Prefectures fetched correctly', () => {
	render(thisComponent);
	const x = screen.getByText(/Number of Year/i);
	expect(x).toBeInTheDocument();
});
