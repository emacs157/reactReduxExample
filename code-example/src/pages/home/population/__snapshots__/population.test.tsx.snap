// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    data-highcharts-chart="0"
    style="overflow: hidden;"
  >
    <div
      class="highcharts-container "
      dir="ltr"
      id="highcharts-z9dhb26-0"
      style="position: relative; overflow: hidden; width: 600px; height: 400px; text-align: left; line-height: normal; z-index: 0; user-select: none; outline: none;"
    >
      <svg
        class="highcharts-root"
        height="400"
        style="font-family: \\"Lucida Grande\\", \\"Lucida Sans Unicode\\", Arial, Helvetica, sans-serif; font-size: 12px;"
        version="1.1"
        viewBox="0 0 600 400"
        width="600"
        xmlns="http://www.w3.org/2000/svg"
      >
        <desc>
          Created with Highcharts 10.0.0
        </desc>
        <defs>
          <clippath
            id="highcharts-z9dhb26-1-"
          >
            <rect
              fill="none"
              height="NaN"
              width="NaN"
              x="0"
              y="0"
            />
          </clippath>
          <clippath
            id="highcharts-z9dhb26-20-"
          >
            <rect
              fill="none"
              height="NaNNaNNaN"
              width="NaNNaNNaN"
              x="0"
              y="0"
            />
          </clippath>
          <clippath
            id="highcharts-z9dhb26-21-"
          >
            <rect
              class="highcharts-animating"
              fill="none"
              height="NaN"
              width="NaN"
              x="0"
              y="0"
            />
          </clippath>
          <clippath
            id="highcharts-z9dhb26-22-"
          >
            <rect
              fill="none"
              height="400"
              width="99"
              x="-99"
              y="-10"
            />
          </clippath>
        </defs>
        <rect
          class="highcharts-background"
          fill="#ffffff"
          height="400"
          rx="0"
          ry="0"
          width="600"
          x="0"
          y="0"
        />
        <rect
          class="highcharts-plot-background"
          fill="none"
          height="NaN"
          width="NaN"
          x="NaN"
          y="10"
        />
        <g
          class="highcharts-grid highcharts-xaxis-grid"
          data-z-index="1"
        >
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke-dasharray="none"
          />
        </g>
        <g
          class="highcharts-grid highcharts-yaxis-grid"
          data-z-index="1"
        >
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke="#e6e6e6"
            stroke-dasharray="none"
            stroke-width="1"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke="#e6e6e6"
            stroke-dasharray="none"
            stroke-width="1"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke="#e6e6e6"
            stroke-dasharray="none"
            stroke-width="1"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke="#e6e6e6"
            stroke-dasharray="none"
            stroke-width="1"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke="#e6e6e6"
            stroke-dasharray="none"
            stroke-width="1"
          />
          <path
            class="highcharts-grid-line"
            d="M 0 0"
            data-z-index="1"
            fill="none"
            opacity="1"
            stroke="#e6e6e6"
            stroke-dasharray="none"
            stroke-width="1"
          />
        </g>
        <rect
          class="highcharts-plot-border"
          data-z-index="1"
          fill="none"
          height="0"
          width="0"
          x="0"
          y="10"
        />
        <g
          class="highcharts-axis highcharts-xaxis"
          data-z-index="2"
        >
          <text
            class="highcharts-axis-title"
            data-z-index="7"
            style="color: rgb(102, 102, 102); fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
            y="-9999"
          >
            Number of Year
          </text>
          <path
            class="highcharts-axis-line"
            d="M 0 0"
            data-z-index="7"
            fill="none"
            stroke="#ccd6eb"
            stroke-width="1"
          />
        </g>
        <g
          class="highcharts-axis highcharts-yaxis"
          data-z-index="2"
        >
          <text
            class="highcharts-axis-title"
            data-z-index="7"
            style="color: rgb(102, 102, 102); fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0) rotate(270 0 -9999)"
            x="0"
            y="-9999"
          >
            Number of Population
          </text>
          <path
            class="highcharts-axis-line"
            d="M 0 0"
            data-z-index="7"
            fill="none"
          />
        </g>
        <g
          class="highcharts-series-group"
          data-z-index="3"
        >
          <g
            class="highcharts-series highcharts-series-0 highcharts-line-series highcharts-color-0"
            clip-path="url(#highcharts-z9dhb26-21-)"
            data-z-index="0.1"
            opacity="1"
            transform="translate(0,10) scale(1 1)"
          >
            <path
              class="highcharts-graph"
              d="M -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000"
              data-z-index="1"
              fill="none"
              stroke="#7cb5ec"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
            <path
              class="highcharts-tracker-line"
              d="M -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000"
              data-z-index="2"
              fill="none"
              stroke="rgba(192,192,192,0.002)"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="22"
              visibility="visible"
            />
          </g>
          <g
            class="highcharts-markers highcharts-series-0 highcharts-line-series highcharts-color-0 highcharts-tracker"
            clip-path="url(#highcharts-z9dhb26-22-)"
            data-z-index="0.1"
            opacity="1"
            transform="translate(0,10) scale(1 1)"
          />
          <g
            class="highcharts-series highcharts-series-1 highcharts-line-series highcharts-color-1"
            clip-path="url(#highcharts-z9dhb26-21-)"
            data-z-index="0.1"
            opacity="1"
            transform="translate(0,10) scale(1 1)"
          >
            <path
              class="highcharts-graph"
              d="M -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000"
              data-z-index="1"
              fill="none"
              stroke="#434348"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
            <path
              class="highcharts-tracker-line"
              d="M -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000"
              data-z-index="2"
              fill="none"
              stroke="rgba(192,192,192,0.002)"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="22"
              visibility="visible"
            />
          </g>
          <g
            class="highcharts-markers highcharts-series-1 highcharts-line-series highcharts-color-1 highcharts-tracker"
            clip-path="url(#highcharts-z9dhb26-22-)"
            data-z-index="0.1"
            opacity="1"
            transform="translate(0,10) scale(1 1)"
          />
          <g
            class="highcharts-series highcharts-series-2 highcharts-line-series highcharts-color-2"
            clip-path="url(#highcharts-z9dhb26-21-)"
            data-z-index="0.1"
            opacity="1"
            transform="translate(0,10) scale(1 1)"
          >
            <path
              class="highcharts-graph"
              d="M -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000"
              data-z-index="1"
              fill="none"
              stroke="#90ed7d"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
            <path
              class="highcharts-tracker-line"
              d="M -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000 L -100000 -100000"
              data-z-index="2"
              fill="none"
              stroke="rgba(192,192,192,0.002)"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="22"
              visibility="visible"
            />
          </g>
          <g
            class="highcharts-markers highcharts-series-2 highcharts-line-series highcharts-color-2 highcharts-tracker"
            clip-path="url(#highcharts-z9dhb26-22-)"
            data-z-index="0.1"
            opacity="1"
            transform="translate(0,10) scale(1 1)"
          />
        </g>
        <text
          class="highcharts-title"
          data-z-index="4"
          style="color: rgb(51, 51, 51); font-size: 18px; fill: #333333;"
          text-anchor="middle"
          x="300"
          y="24"
        >
          codeCheck
        </text>
        <text
          class="highcharts-subtitle"
          data-z-index="4"
          style="color: rgb(102, 102, 102); fill: #666666;"
          text-anchor="middle"
          x="300"
          y="10"
        >
          homeWork
        </text>
        <text
          class="highcharts-caption"
          data-z-index="4"
          style="color: rgb(102, 102, 102); fill: #666666;"
          text-anchor="start"
          x="10"
          y="397"
        />
        <g
          class="highcharts-legend highcharts-no-tooltip"
          data-z-index="7"
          transform="translate(300,359)"
        >
          <rect
            class="highcharts-legend-box"
            fill="none"
            rx="0"
            ry="0"
            visibility="visible"
          />
          <g
            data-z-index="1"
          >
            <g>
              <g
                class="highcharts-legend-item highcharts-line-series highcharts-color-0 highcharts-series-0"
                data-z-index="1"
                transform="translate(8,3)"
              >
                <path
                  class="highcharts-graph"
                  d="M 0 11 L 16 11"
                  fill="none"
                  stroke="#7cb5ec"
                  stroke-width="2"
                />
                <path
                  class="highcharts-point"
                  d="M 8 15 A 4 4 0 1 1 8.003999999333336 14.999998000000167 Z"
                  fill="#7cb5ec"
                  opacity="1"
                />
                <text
                  data-z-index="2"
                  style="color: rgb(51, 51, 51); cursor: pointer; font-size: 12px; font-weight: bold; fill: #333333;"
                  text-anchor="start"
                  x="21"
                  y="15"
                >
                  栃木県
                </text>
              </g>
              <g
                class="highcharts-legend-item highcharts-line-series highcharts-color-1 highcharts-series-1"
                data-z-index="1"
                transform="translate(0,3)"
              >
                <path
                  class="highcharts-graph"
                  d="M 0 11 L 16 11"
                  fill="none"
                  stroke="#434348"
                  stroke-width="2"
                />
                <path
                  class="highcharts-point"
                  d="M 8 7 L 12 11 L 8 15 L 4 11 Z"
                  fill="#434348"
                  opacity="1"
                />
                <text
                  data-z-index="2"
                  style="color: rgb(51, 51, 51); cursor: pointer; font-size: 12px; font-weight: bold; fill: #333333;"
                  text-anchor="start"
                  x="21"
                  y="15"
                >
                  高知県
                </text>
              </g>
              <g
                class="highcharts-legend-item highcharts-line-series highcharts-color-2 highcharts-series-2"
                data-z-index="1"
                transform="translate(0,3)"
              >
                <path
                  class="highcharts-graph"
                  d="M 0 11 L 16 11"
                  fill="none"
                  stroke="#90ed7d"
                  stroke-width="2"
                />
                <path
                  class="highcharts-point"
                  d="M 4 7 L 12 7 L 12 15 L 4 15 Z"
                  fill="#90ed7d"
                  opacity="1"
                />
                <text
                  data-z-index="2"
                  style="color: rgb(51, 51, 51); cursor: pointer; font-size: 12px; font-weight: bold; fill: #333333;"
                  text-anchor="start"
                  x="21"
                  y="15"
                >
                  宮崎県
                </text>
              </g>
            </g>
          </g>
        </g>
        <g
          class="highcharts-axis-labels highcharts-xaxis-labels"
          data-z-index="7"
        >
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            1960
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            1965
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            1970
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            1975
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            1980
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            1985
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            1990
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            1995
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2000
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2005
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2010
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2015
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2020
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2025
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2030
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2035
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2040
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="middle"
            transform="translate(0,0)"
            x="0"
          >
            2045
          </text>
        </g>
        <g
          class="highcharts-axis-labels highcharts-yaxis-labels"
          data-z-index="7"
        >
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="end"
            transform="translate(0,0)"
            x="0"
          >
            0
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="end"
            transform="translate(0,0)"
            x="0"
          >
            500k
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="end"
            transform="translate(0,0)"
            x="0"
          >
            1 000k
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="end"
            transform="translate(0,0)"
            x="0"
          >
            1 500k
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="end"
            transform="translate(0,0)"
            x="0"
          >
            2 000k
          </text>
          <text
            style="color: rgb(102, 102, 102); cursor: default; font-size: 11px; fill: #666666;"
            text-anchor="end"
            transform="translate(0,0)"
            x="0"
          >
            2 500k
          </text>
        </g>
        <text
          class="highcharts-credits"
          data-z-index="8"
          style="cursor: pointer; color: rgb(153, 153, 153); font-size: 9px; fill: #999999;"
          text-anchor="end"
          x="590"
          y="395"
        >
          Highcharts.com
        </text>
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
